examples/todo/src/datasources/db.datasource.ts

```diff
@@ -3,22 +3,17 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import * as path from 'path';
-// The juggler reference must exist for consuming code to correctly infer
-// type info used in the "db" export (contained in juggler.DataSource).
-// tslint:disable-next-line:no-unused-variable
-import {juggler} from '@loopback/repository';
+import {inject} from '@loopback/core';
+import {juggler, DataSource} from '@loopback/repository';
+const config = require('./db.datasource.json');
```

Comment:
I keep getting an error when I do this as follows: `Type 'typeof FILENAME' is not assignable to type 'DataSource'` --- and the new import from JSON in TS is in 2.9 but we had to revert back to 2.8. This will have to be addressed in a follow up PR once we can use TS 2.9